docs/spec/interfaces.md

```diff
@@ -15,33 +15,15 @@ can direct to their configured destination.
 
 A **Subscribable** resource may be referenced in the _from_ field of a
 _Subscription_. The _Subscribable_ resource MUST expose a
-_status.subscribable.channelable_ field (an _ObjectReference_). The resource
-referenced in the _status.subscribable.channelable_ field MUST be a
-_Channelable_ resource; the field MAY refer back to this _Subscribable_ as a
+_status.subscriptions_ field (an _ObjectReference_). The resource
+referenced in the _status.subscriptions_ field MUST be a
+_ChannelSubscriptionSet_ resource; the field MAY refer back to this _Subscribable_ as a
```

Comment:
I'm confused about the self-referencing aspect. Is _ChannelSubscriptionSet_ also _Subscribable_?

Reply:
Yea the self referencing bit is confusing, that's a holdover that I should have removed.

I also proposed that the Subscribable interface be removed in #4, but I'll update this PR to make more sense individually.